lantern/oop/homework/homework.py

```diff
@@ -1,26 +1,59 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
+
+from typing import List
+
+
 class Cat:
     """
     Write Class Cat which will receive age from user
+
     * Add to class average_speed variable which will get it's values
       from private method _set_average_speed()
 
     * Add to class saturation_level variable with value 50
+    """
 
-    * Implement private methods _increase_saturation_level and _reduce_saturation_level
-      that will receive value and add/subtract from saturation_level this value
-      if saturation_level is less than 0, return 0
-      if saturation_level is grosser than 100, return 100
+    def __init__(self, age):
+        self.age = age
+        self.average_speed = self._set_average_speed()
+        self.saturation_level = 50
+
+    """
+    * Implement method eat wh+ich will receive from user product value
+      if product == fodder use _increase_saturation_level with value == 10
+      if product == apple use _increase_saturation_level with value == 5
+      if product == milk use _increase_saturation_level with value == 2
+     """
```

Comment:
please write a comment related to function inside the function, after defenition